### PR TITLE
Add isVerifiedArtist to response object

### PR DESCRIPTION
**Pull request by ChunkyProgrammer**

### CODE REVIEW

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

File: src/Youtube-Scraper.js [hunk 1]
```diff
@@ -130,7 +132,8 @@ class YoutubeScraper {
             video_entry.lengthSeconds = this.calculate_length_in_seconds(video_entry.timeText);
             video_entry.videoThumbnails = this.extract_thumbnail_data(video_entry.videoId);
             if ('ownerBadges' in videoRenderer) {
-                video_entry.isVerified = (videoRenderer.ownerBadges[0].metadataBadgeRenderer.tooltip == 'Verified')
+                video_entry.isVerified = (videoRenderer.ownerBadges[0].metadataBadgeRenderer.style === 'BADGE_STYLE_TYPE_VERIFIED')
+                video_entry.isVerifiedArtist = (videoRenderer.ownerBadges[0].metadataBadgeRenderer.style === 'BADGE_STYLE_TYPE_VERIFIED_ARTIST')
```
Comment (PikachuEXE) [inline src/Youtube-Scraper.js:136]: Possible to add reference to documentation if any?
If there is no documentation I will just approve it

**Review by peepopoggers — DISMISSED** [review]
LGTM (I am presuming we will handle this new variable in FT after merge?)

File: src/Youtube-Scraper.js (hunk 1 above)
Comment (PikachuEXE) [reply src/Youtube-Scraper.js:136]: Just as a reference
I saw example code from another project
https://github.com/fent/node-ytdl-core/blob/master/example/info.json

**Review by PikachuEXE — DISMISSED** [review]
Example code seen in
https://github.com/fent/node-ytdl-core/blob/master/example/info.json

**Comment by PikachuEXE** [comment]
Is auto merge enabled and setup properly?

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
No something is wrong with automerge on this repo.

**Comment by peepopoggers** [comment]
@PrestonN

**Comment by PrestonN** [comment]
This PR will need to be redone and merged against the Development branch instead of the Master branch. There are different rules for merging into master which is why the auto merge isn't working. Please make this change for your PR and then it should work.

**Review by efb4f5ff-1298-471a-8973-3d47447115dc — APPROVED** [review]

**Review by peepopoggers — APPROVED** [review]

**Review by PikachuEXE — APPROVED** [review]

**Pull request merged**